### PR TITLE
Add rate-limit header details

### PR DESCRIPTION
**Pull request by shyam-habarakada**
Getting ready to enable rate-limiting in production. This has already been in place in the staging environment and this introduces additional documentation to support the `X-RateLimit-*` headers we’ve added to support better handling of throttling.

See https://github.com/10Kft/10kft-api/blob/v1-overhaul/sections/first-things-first.md for the rendered markdown view of the page that contains the bulk of this change.

### CODE REVIEW

**Comment by shyam-habarakada** [comment]
[[INFRA] Configure memcached and implment rate-limitting in production](https://trello.com/c/9gLwsHus/1421-infra-configure-memcached-and-implment-rate-limitting-in-production)

**Comment by shyam-habarakada** [comment]
[[INFRA] Configure memcached and implment rate-limitting in production](https://trello.com/c/9gLwsHus/1421-infra-configure-memcached-and-implment-rate-limitting-in-production)

File: sections/first-things-first.md [hunk 1]
```diff
@@ -128,40 +128,55 @@ Respond with standard HTTP error messages, with 4XX codes to indicate client (ca
...
 
-### What to do when being throttled?
+When an application exceeds the rate limit, the API will return a standard HTTP `429 Too Many Requests` response.
 
-Our current recommendation is to implement a progressive back-off approach until your requests start processing normally. The psuedo-code below illustrate a simple scenario which you may adopt to suit your particular case.
+### How to handle being throttled
+
+The specific details of rate limit policies in effect may vary without notice. Application developers should implement detection and handling of rate-limiting behavior using `X-Ratelimit-*` headers described below.
+
+When a request from your application receives a `429 Too many Requests` response, you can check the returned HTTP headers to see your current rate limit status:
```
Comment (swordfish444) [inline sections/first-things-first.md:141]: @shyam-habarakada This reads as if the rate limit headers will only show up on the 429 response. I feel we should remove this sentence.

**Comment by swordfish444** [comment]
@shyam-habarakada One comment for you to review. Otherwise LGTM!

File: sections/first-things-first.md (hunk 1 above)
Comment (shyam-habarakada) [reply sections/first-things-first.md:141]: That is actually the case. Some APIs (e.g. Twitter) exposes this on all requests, but there are some practical and theoretical problems with trying to do that. And for real-world scenarios I think an application is only going to care about this when they have hit the limit, so what we have here is intentional. We can discuss in person if needed.

**Pull request merged**